allennlp_models/modelcards/nlvr2-vilbert-head.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "nlvr2-vilbert",
+    "registered_model_name": "nlvr2",
+    "registered_predictor_name": null,
+    "display_name": "Visual Entailment - NLVR2",
+    "task_id": "nlvr2",
+    "model_details": {
+        "description": "This model is based on the ViLBERT architecture. The image features are obtained using the ResNet backbone and Faster RCNN (region detection).",
+        "short_description": "ViLBERT-based model for Visual Entailment.",
```

Comment:
I think it's worth noting here how `nlvr2-vilbert-head` is different from `nlvr2-vilbert`

Reply:
Done!